Revision: lvm--dilinger--2--patch-2
Archive: dilinger@voxel.net--2004
Creator: Andres Salomon <dilinger@voxel.net>
Date: Mon May  3 01:56:17 EDT 2004
Standard-date: 2004-05-03 05:56:17 GMT
Modified-files: make.tmpl.in
New-patches: dilinger@voxel.net--2004/lvm--dilinger--2--patch-2
Summary: Clean fix..
Keywords: 

* added files

    {arch}/lvm/lvm--dilinger/lvm--dilinger--2/dilinger@voxel.net--2004/patch-log/patch-2

* modified files

--- orig/make.tmpl.in
+++ mod/make.tmpl.in
@@ -167,6 +167,7 @@
 clean: $(SUBDIRS.clean)
 	$(RM) $(OBJECTS) $(TARGETS) $(CLEAN_TARGETS) $(SOURCES:%.c=%.d) \
 	      $(SOURCES:%.c=%.pot) $(LDDEPS)
+	$(RM) lvm.o
 
 distclean: $(SUBDIRS.distclean)
 	$(RM) $(OBJECTS) $(TARGETS) $(CLEAN_TARGETS) $(SOURCES:%.c=%.d) \



